integration/emr/alluxio-emr.sh

```diff
@@ -143,9 +143,12 @@ emr_install_alluxio() {
   download_file "${alluxio_tarball}"
 
   release=$(basename "${alluxio_tarball}")
-  local release_unzip
-  release_unzip="${release%%-*}" # trims everything after the first '-', ex. alluxio-foo-bar-whatever -> alluxio
-
+  local release_prefix
+  if [[ "${release}" == *-* ]]; then
+    release_unzip="${release%%-*}" # trims everything after the first '-', ex. alluxio-foo-bar-whatever -> alluxio
```

Comment:
both of these expressions seem very fragile to me :/
the original expression of converting alluxio-* -> alluxio seems unnecessarily difficult

is it not possible to trim the ".tar.gz" suffix from the tarball and use that as the directory name that the tarball unzips to?

Reply:
emmm... the problem is the unzipped folder from  <value>.tar.gz doesn't name <name>